scripts/check-cpu-aes.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# zenoss-inspector-info
+# zenoss-inspector-tags cpu os
+
+cat /proc/cpuinfo | grep -E "flags.*:.*aes"
```

Comment:
In the future, consider splitting this into 2 scripts - one to collect the data and other to analyze the data.
This way, we would have the contents of /proc/cpuinfo available for review for other possible issues unrelated to this AES issue.